chat/rich-links/src/RichLink/RichLink.tsx

```diff
@@ -62,9 +64,8 @@ export const RichLink = forwardRef<HTMLAnchorElement, RichLinkProps>(
           [imageBackgroundStyles(imageUrl ?? '')]: showImageBackground,
         })}
         as="a"
-        href={href}
-        target="_blank"
-        {...anchorProps}
+        // Cast to div props to get around Card's Box typing
```

Comment:
Can you add a link here, in the comment, to the Jira ticket you created about this?